lib/wallet/src/Cardano/Wallet/Read/Primitive/Tx/Features/Fee.hs

```diff
@@ -15,9 +13,8 @@ import Cardano.Wallet.Read.Eras
 import Cardano.Wallet.Read.Tx.Fee
     ( Fee (..), FeeType )
 
-import qualified Cardano.Ledger.Coin as SL
 import qualified Cardano.Wallet.Primitive.Types.Coin as W
-import qualified Cardano.Wallet.Primitive.Types.Coin as Coin
+import qualified Cardano.Wallet.Shelley.Compatibility.Ledger as Ledger
```

Comment:
The plan was to inline `*.Compatibility` into these more specialized modules — but this is still good enough for merging.